stetho/src/main/java/com/facebook/stetho/inspector/database/DefaultDatabaseFilesProvider.java

```diff
@@ -32,6 +34,45 @@ public DefaultDatabaseFilesProvider(Context context) {
     for (String filename : mContext.databaseList()) {
       databaseFiles.add(new File(filename));
     }
+
+    FilenameFilter filenameFilter = new FilenameFilter() {
+      @Override
+      public boolean accept(File dir, String filename) {
+        return filename.endsWith(".sql")
+            || filename.endsWith(".sqlite")
+            || filename.endsWith(".db");
+      }
+    };
+
+    //Add External Databases if Present
+    if(isExternalAvailable()){
+      final File absoluteFile = mContext.getExternalFilesDir(null).getAbsoluteFile();
```

Comment:
I'm uncomfortable with this being the default.  These files have dubious security implications and are not necessarily accessible without special permissions pre-KitKat.  Perhaps we can make this an option to the DefaultDatabaseFilesProvider and use our extensibility API (which is being improved BTW) to let callers tweak how they want this to be brought in?